test/test_wifi_dto/test_wifi_dto.cpp

```diff
@@ -6,23 +6,24 @@
 #include "WiFiDTO.h"
 
 /**
- * @brief Test WiFIDTO Class Serialization
+ * @brief 
+ * Test WiFIDTO Class Serialization 
+ * This test in in order to show and proof how to use serialized and deserialized 
+ * methods inside DTO class.
  * 
  */
-void test_serialized_wifi_dto(void) {
-     wifi::DTOConfig wifi_params;  //to serialized object 
+void test_serialized_and_deserialized_wifi_dto(void) {
```

Comment:
At the end the name could be ``test_serialized_and_deserialized_dto_config`, remember that `WiFiDTO` was renamed. Also this applies to the test file name.